verify_consolidated.py

```
@@ -0,0 +1,31 @@
+    page.wait_for_timeout(10000)
+
+    # Take dashboard screenshot
+    page.screenshot(path="dashboard_final.png")
+
+    # Try to navigate to Wallet (simpler page)
+    try:
+        page.click("text=Wallet")
+        page.wait_for_load_state("networkidle")
+        page.wait_for_timeout(3000)
+        page.screenshot(path="wallet_final.png")
+    except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)